### PR TITLE
Added correct payload converter for planting site creation

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#2736** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2736?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/utils/draftPlantingSiteUtils.ts [hunk 1]
```diff
@@ -82,3 +88,31 @@ export const toDraft = (payload: DraftPlantingSitePayload): DraftPlantingSite =>
...
+    name,
+    organizationId,
+    plantingSeasons,
+    plantingZones,
+    projectId,
+    timeZone,
+  };
+
+  return payload;
+};
```
Comment (joshbuchea) [inline src/utils/draftPlantingSiteUtils.ts:118]: Are there additional fields aside from the ones destructed here? Is that why this doesn't just `return { ...site };`?

**Review by joshbuchea — APPROVED** [review]

**Review by nickgraz — APPROVED** [review]
Straightforward enough!

File: src/utils/draftPlantingSiteUtils.ts (hunk 1 above)
Comment (tommylau523) [reply src/utils/draftPlantingSiteUtils.ts:118]: Mostly wanted to respect the API payload. Should be okay in this instance.

**Pull request merged**